app/models/listing.rb

```diff
@@ -17,6 +17,22 @@ class Listing < ApplicationRecord
   enum state: { received: 0, fulfilled: 1 }
 
   def name
-    "#{type}: #{tags}"
+    "#{type}: #{tags.join(', ')}"
```

Comment:
I think this needs to be `all_tags_list.join`
I recently added some methods like this also, tho not duplicative. Still, prob should review next week (i'll add an issue).